testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/model/ReadFullModelTestCase.java

```diff
@@ -26,63 +26,54 @@
 
-import org.jboss.arquillian.container.test.api.ContainerController;
-import org.jboss.arquillian.container.test.api.RunAsClient;
-import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.arquillian.test.api.ArquillianResource;
+import javax.inject.Inject;
+
 import org.jboss.as.controller.PathAddress;
-import org.jboss.as.controller.client.ModelControllerClient;
 import org.jboss.as.controller.operations.common.Util;
-import org.jboss.as.model.test.ModelTestUtils;
-import org.jboss.as.test.shared.ServerReload;
-import org.jboss.as.test.shared.TestSuiteEnvironment;
 import org.jboss.dmr.ModelNode;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.wildfly.core.testrunner.ManagementClient;
+import org.wildfly.core.testrunner.ServerControl;
+import org.wildfly.core.testrunner.ServerController;
+import org.wildfly.core.testrunner.WildflyTestRunner;
 
 /**
  * Ensures that the full model including runtime resources/attributes can be read in both normal and admin-only mode
  * for the fullest server config possible
  *
  * @author Kabir Khan
+ * @author Tomaz Cerar
  */
-@RunWith(Arquillian.class)
-@RunAsClient
+@ServerControl(manual = true)
+@RunWith(WildflyTestRunner.class)
 public class ReadFullModelTestCase {
 
     //This is the full-ha setup which is the fullest config we have
     //Reuse this rather than a setup
-    public static final String SERVER = "jbossas-messaging-ha-server1";
-
-    @ArquillianResource
-    protected static ContainerController controller;
+    private static final String SERVER_CONFIG = "standalone-full-ha.xml";
 
-    protected static ModelControllerClient createClient1() {
-        return TestSuiteEnvironment.getModelControllerClient();
-    }
+    @Inject
+    private ServerController container;
 
 
     @Test
     public void test() throws Exception {
-        controller.start(SERVER);
+        container.start(SERVER_CONFIG, true);
         try {
-            ModelControllerClient client = TestSuiteEnvironment.getModelControllerClient();
+            ManagementClient client = container.getClient();
             ModelNode rr = Util.createEmptyOperation(READ_RESOURCE_OPERATION, PathAddress.EMPTY_ADDRESS);
             rr.get(INCLUDE_RUNTIME).set(true);
             rr.get(RECURSIVE).set(true);
-            ModelNode result = ModelTestUtils.checkResultAndGetContents(client.execute(rr));
+            ModelNode result = client.executeForResult(rr);
 
             //Just a quick sanity test to check that we are full-ha by making sure some of the expected subsystems are there
             Assert.assertTrue(result.hasDefined(SUBSYSTEM, "messaging-activemq"));
             Assert.assertTrue(result.hasDefined(SUBSYSTEM, "jgroups"));
-
-            ServerReload.executeReloadAndWaitForCompletion(client, true);
-            result = ModelTestUtils.checkResultAndGetContents(client.execute(rr));
+            client.executeForResult(rr);
```

Comment:
This test no longer tests both normal and admin-only. Before it started normal and reloaded to admin only, now it starts admin-only, doesn't reload and the executes the read op again.